content/blog/2023/08/28/2023-08-28-renovate-bote-probe-blog.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Renovate Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/renovate-probe.jpg
+sig:
+- gsoc
+---
+
+== Introduction
+
+Bots are used to automate tasks in a plugin. Plugin Health Scoring Systems tracks usage of two bots - Dependabot, link:https://docs.renovatebot.com/[Renovate].
+
+Renovate probe checks whether Renovate is configured in a plugin.
```

Comment:
```suggestion
Bots are used to automate tasks in a plugin.
The Plugin Health Scoring System tracks usage of two bots - link:https://github.com/dependabot[Dependabot] and link:https://docs.renovatebot.com/[Renovate].

The renovate probe checks whether Renovate is configured in a plugin.
```

I would suggest adding some context here, maybe what Dependabot and Renovate do and why the probe is necessary/desired